presto-docs/src/main/sphinx/release/release-320.rst

```diff
@@ -0,0 +1,66 @@
+===========
+Release 320
+===========
+
+General Changes
+---------------
+
+* Fix incorrect parameter binding order when parameters appear inside a
+  ``WITH`` clause. (:issue:`1191`)
+* Fix planning failure for certain queries involving a mix of ``OUTER``
```

Comment:
Remove the code quotes since this is referring to the logical join type and not the specific SQL syntax